vsts_ci/darwin/continuous-build-darwin.yml

```diff
@@ -12,5 +12,15 @@ steps:
     displayName: "Update and install required tools"
 
   - script: |
-      sudo -E `which tox`
-    displayName: "Run tests against iotedgehubdev source code with tox"
+      brew install docker
+      brew install docker-machine
+      brew install docker-machine-driver-xhyve
+      sudo chown root:wheel $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+      sudo chmod u+s $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+      docker-machine create --driver xhyve default
+      docker-machine env default
+      eval $(docker-machine env default)
+      brew services start docker-machine
+      brew install docker-compose
+      sudo -E `which tox` -e $(PYTHON_VERSION)
```

Comment:
I just realize there is a `-E` option for `sudo`. I guess with `-E`, you can simply `sudo -E tox`. Please help verify. Thanks.